src/script.js

```diff
@@ -1,3 +1,15 @@
+import * as jquery from 'jquery';
+// Load phaser (https://github.com/photonstorm/phaser/issues/1974)
+import PIXI from 'expose-loader?PIXI!phaser-ce/build/custom/pixi.js';
+import p2 from 'expose-loader?p2!phaser-ce/build/custom/p2.js';
+import Phaser from 'expose-loader?Phaser!phaser-ce/build/custom/phaser-split.js';
+
+import { Game } from './game';
+
+// Export stuff that needs to be on the window object (Hack)
+window.$j1 = jquery;
```

Comment:
Why $j1 and not one of the default properties it normally exposes?

Reply:
I changed this in a later commit

Reply:
Ahh good! Being a huge commit, I'm going through the individual commits, maybe I should start bottom up instead ;)